### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ghost**
* Update to a later version of Ubuntu that comes with Python 3.8 (same as data standard uses currently)
* Update to latest version of NPM
* NPM Install as normal user not root to avoid permissions problems
* Upgraded bootstrap_build/package-lock.json updates to latest format
* Specify VirtualBox name to avoid problems with some versions if default name is to long
* Altered Readme to make clear that docs writers can use a simpler setup elsewhere
* In line with last point, made just one build comand as build-sphinx should not be usefull any more
* Made instructions for link checking easier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,13 @@ This repository holds the custom theme for https://github.com/openownership/data
 
 It also holds various files and tools to build this theme.
 
-## To use as a developer / writer
+## To use to write docs
```
Comment (siwhitehouse) [inline README.md:7]: I think we could be more directive and simply say:

"If you are writing docs then please follow the instructions at https://github.com/openownership/data-standard/blob/master/README.md"

Comment (ghost) [reply README.md:7]: I think it at least needs:

"If you are writing docs only and **don't** need to change the theme then please follow the instructions at https://github.com/openownership/data-standard/blob/master/README.md"

Comment (ghost) [reply README.md:7]: Checked on call with Kadie, we'll go with what I suggested.

**Comment by ghost** [comment]
Simon and Kadie has seen new section in  README - now asking a dev person to review tech bits

**Review by rhiaro — APPROVED** [review]

**Comment by ghost** [comment]
Restested

**Pull request merged**